ingenialink/canopen/register.py

```diff
@@ -82,3 +82,8 @@ def idx(self):
     def subidx(self):
         """int: Register subindex."""
         return self.__subidx
+
+    @property
+    def mapped_address(self):
+        """int: Register mapped address used for monitoring/disturbance."""
+        return self.idx
```

Comment:
the mapped address does not change with the subnode?

Reply:
No, it's the same as the idx